### ISSUE TITLE
future  Balance Sheet and Profit and Loss Account

### ISSUE THREAD

**Issue by BorderITS** [borderitsolutions/amadaa-server#27]
Balance Sheet and Profit and Loss Account
Jul 20, 2014111,514 views15 Likes6 CommentsShare on LinkedInShare on FacebookShare on Twitter
In this article we will also discuss:

Balance Sheet
Trading and Profit and Loss Account
Adjustments of Final Accounts
While introducing the subject of accounting to the readers (In Chapter I) it was stated that one of the main objectives of financial accounting is to provide information about the profits earned or loss suffered by the business during a particular period.

Previously while discussing the basic accounting equation it was noted that A - L = P, where A represents assets (property and possession) owned by the business; L represents liabilities (claims against the business of the creditors) and P represents the proprietor's funds (equity) in the business.,

Accounting Concept of Income

The concept of 'income' is different to the economists and accountants. Economists concept of income is that of 'real income' meaning thereby the increase-in real terms of the ownership funds between two points of time.

In accounting the term income is known as 'net profit'. It was stated earlier :

Sales - Merchandising cost = Gross profit and Gross profit - Expenses of doing business = Net profit

In other words, Revenue - Expenses = Net profit.

These terms are explained below:

Revenue

It is the monetary value of the products sold or services rendered to the customers during the period. It results from sales services and source like interest, dividend and commission etc. For example, sales affected by the business and charge made for services rendered by the business constitutes revenue. However, all cash receipt may nott be revenue. 

Thus, money borrowed leads to cash receipt but it does not constitute revenue. Similarly additional capital brought in increases proprietor's funds but it is not revenue.

Expenses/Cost of (doing business)

Expenditure incurred by the business to earn revenue is termed as expense or cost of doing business. Examples of expenses are raw materials consumed, salaries, rent, depreciation, advertisement etc.

Cash v/s Accrual Basis of Accounting

Small business, individual professionals and non-trading concerns usually adopt cash basis of accounting. Under this system, incomes are considered to have been earned only when received ill cash and expenses are considered to have been incurred only actually paid. Hence, under this system the profit or loss of an accounting period is the difference between incomes received and the expenses paid. Though the cash basis of accounting is simple (no adjustment is required) but il loses its comparability.

Under accrual basis all incomes are credited to the period in which earned irrespective of the fact whether received or not. Similarly, all expenses are debited to the period in which incurred irrespective of the fact whether paid or not. It is a scientific basis of accounting, though a bit difficult.

Matching Concept. Requires that expenses should be matched to the revenues of the appropriate accounting period. So we must determine what are the revenues earned during a particular accounting period and the expenses incurred to earn these revenues.
It is the matching concept which justifies accrual basis of accounting.

Accruals and Deferrals

Accounting is expected to measure or ascertain the net income of the business during the accounting period. Normally, it is the calendar year (1st January to 31st December) but in other cases it may be Financial year (Ist April to 31st March) or any other period according to the convention of the business community of the area.

The combined impact of matching concept and the accounting period concept on accounting has resulted in accruals and deferrals.

Accrued or Outstanding expenses

It is the term which denotes that expenditure has been incurred during the accounting period but the same has not been paid in cash e.g. Salary, Rent, Wage etc. becoming due but not paid.

Deferred or pre-paid expenses

It is the term which denotes that payment in cash has been made "in advance but the full benefit of this payment has not been reaped by the current accounting period, e.g., Insurance paid in advance.

Accrued or outstanding Income

It is the term which denotes that the income has been earned but the cash has not been received against the same. Income has accrued due but not received e.g. Interest on investments etc. 

Differed or Received in-advance Income

It is the term which denotes income which has been received (in cash) in advance but it has not been earned so far e.g. rent received in advance. All the accruals and deferrals arc not be adjusted at the end of the accounting period (end-period adjustments) in order to find out the income of the business during the period under review. The procedure of ascertaining (i) business income and (ii) financial position is being described, in detail below:

In fact, these arc two most important of many objectives of book-keeping. In order to know the profits earned by him he prepares a trading and loss account and in order to know the financial position of his business on the last day of the financial period he prepares a balance sheet.

Such accounts are called 'Final Accounts'. Preparation of final accounts is the concluding step of accounting cycle. In fact, final accounts include a number of accounts (i) Manufacturing/ Production account, (ii) Trading account, (iii) Profit and loss account and (iv) Balance sheet.

Practically balance sheet is a statement but for accounting purposes here it is treated as a part vital accounts.

The preparation of above all or any of the above accounts depends upon the nature of the business being carried on by the business concerned. In case of a manufacturing business manufacturing account, trading account, profit and loss account and balance sheet form the parts of final accounts whereas in case of trading business all other accounts are prepared with the exception of manufacturing account. Each of these accounts provide a specific vital information to businessman to help to control and organize the business activities in a batter way.

Balance Sheet

Balance sheet is a statement of financial position of a concern at a given date. It shows the financial position of a concern at a given date of accounting period, because the situation may be entirely different on the following day and indeed, might have been quite different a day earlier.

A balance sheet may, therefore, be defined as "a statement prepared with a view to measure the exact financial position of a business on a certain date.
"It is prepared from the trial balance after all the balances of nominal accounts are transferred to trading and profit and loss account and corresponding accounts in the ledger are closed. The balances now left in the trial balance are either personal or real accounts. In other words, they either represent assets or liabilities existing on the date of closing of accounts.

All these assets and liabilities are displayed in the balance sheet according to certain principles such as :

(a) All real and personal account having debit balances should be shown on the assets side of balance sheet which is on the right-hand side.
(b) All the real and personal account having credit balances should be shown on the liabilities side of balance sheet, which is on the left-hand side. The excess of assets over liabilities represents the capital of the owner. This figure of capital must tally with the closing balance of capital account in the ledger after the net profit or loss has been transferred therein.

It shows that when real and personal accounts are placed on the opposite sides of balance sheet according to the nature of balances, the assets side should be equal to liabilities side.

As stated earlier and personal accounts having debit balances are called assets; actually at trader's property and possessions as also the debts owing to him (sundry debtors and bills receivable) are assets.

The real and personal accounts having credit balances along with owner's capital are shown as liabilities. So liabilities are the debts owing by a business to third parties and the owner of the business.

Classification of Assets

Assets have been classified as follows:

(a) Fixed Assets. The assets of a durable nature which are used in business and are acquired and intended to be retained permanently for the purpose of carrying on the business, such as land, building, machinery and furniture etc. They are also sometimes called as capital assets or fixed capital expenditures or long lived assets. Fixed assets are collectively known as 'Block'.

(b) Floating or Circulation Asset. Those temporarily held assets which are meant for resale or which frequently undergo change e.g. cash, stock, stores, debtors and bills receivable. Floating assets are again sub-divided into two parts, liquid assets and non-liquid assets. Liquid assets are those which can be readily converted into cash without appreciable loss. Cash in hand and cash at bank are the example of such assets. Other assets which cannot be readily converted into cash, or not without appreciable loss, are called non-liquid assets e.g., stock, stores.

(c) Fictitious Assets. Those assets which are not represented by anything concrete or tangible. Preliminary expenses, debit balance of profit and loss account are the examples of such assets. These are also called as 'nominal' or 'imaginary' assets.

Classification of Liabilities

The liabilities of a concern can be classified as given below:

(a) Fixed Liabilities. Those liabilities which are to be redeemed after a long period of time. This includes long term loans.

(b) Current Liabilities. Those liabilities which are to be redeemed in near future usually within a year. Trade creditors, bank loan, bills payable etc., are examples of current liabilities.

(c) Contingent Liabilities. These are not actual liabilities but their becoming actual liability is contingent on the happening of a certain event. In other words, they would become liabilities in the future provided the contemplated event occurs. If it does not occur, no liability is incurred. Since such a liability is not an actual liability, it is not shown in the balance sheet. Usually, it is mentioned in the form of a footnote.

Form of Balance Sheet

A balance sheet has two sides-the left-hand side and the right-hand side. These two sides, however, are not comparable with the debit side and credit side of a ledger account because balance sheet is not an account. Words 'To' or 'By' are not used in the balance sheet The left-hand side is liabilities side and contains credit balances of all real and personal accounts and on the right-hand side which is "assets" side, are listed the debit balances of real and personal accounts.

Arrangement of Assets and Liabilities in Balance sheet

The assets and liabilities should be arranged in balance sheet in some specific order. Arrangement of assets and liabilities in the balance sheet is called 'Marshaling of assets and liabilities'. There are two systems of arrangement of assets and liabilities in the balance sheet:

(a) Order of Liquidity.
(b) Order of Permanence.

In liquidity order most easily realizable assets are shown first and are followed by assets which are less easily resalable. So, the assets most difficult of realization will be shown last. In case of liabilities, these will be shown in the order in which they are payable the most pressing liability being placed first.

Distinction between Trial Balance and Balance Sheet

1. Trial balance is the 'means' of accounting process of which the balance sheet is the 'end' because a balance sheet is always prepared from the figures taken out of trial balance.
2. The purpose of preparing a trial balance is to check the arithmetical accuracy of account books; but balance sheet is drafted to reveal the financial position of the business.
3. The two sides of balance sheet are called 'liabilities' and 'assets' sides respectively but in case of -trial balance the columns are 'debit' and 'credit' columns.
4. For completing the accounting cycle, the preparation of balance sheet is. necessary; but the preparation of trial balance is not always necessary. -
5. The period after which a balance sheet is prepared, is normally one year but trial balance is prepared very often and it may be monthly, quarterly or half-yearly.
6. Trial balance contains in it all the three types of accounts viz. personal real and nominal, but balance sheet contains only personal and real accounts.~
7. Generally, trial balance does not contain closing stock but balance sheet does.
8. It is not possible to know the accrued, advance, outstanding and prepaid receipts and expenses from trial balance, but balance sheet discloses such items.

Manufacturing Account

Some concerns like to ascertain the cost of goods manufactured by them during the year distinctly before they prepare the trading account and ascertain the gross profit. This account is called the manufacturing account and is prepared in addition to the trading account. It has the under mentioned characteristics:

(i) Since the purpose of preparation of this account is to ascertain the cost of goods produced during the year, the opening and closing stocks of finished goods are not entered in it ; they will figure in trading account.
(ii) In respect of materials it is the figure of materials consumed which is debited to the account. This figure is obtained by adjusting the purchase of materials for the opening and closing stocks of materials e.g., Opening stock of raw materials Add: purchases of raw materials during the year Less: closing stock of raw materials Cost of materials consumed
(iii) In the manufacturing concern there will always be some unfinished goods or work-in-progress. The cost of work-in-progress at the end of the year is credited to this account, shown in the balance sheet and debited to the manufacturing account of next year as on opening balance.
(iv) All expenses in factory- wages, power and fuel, repairs and maintenance, factory salaries factory rent and rates are debited to this account. Depreciation on machinery is also .debited to this account and not to the profit and loss account as is usually done.
(v) Amounts raised by sale of waste or scrap materials are deducted from raw material purchases.
(vi) Now the difference is two sides of this account will be the cost of goods manufactured during the year. This cost will be credited to manufacturing account and debited to trading account.

The trading account will now comprise only the opening and closing stock of finished goods, the cost of goods manufactured as transferred from manufacturing account and sales of finished goods. The gross profit will be transferred to profit & loss account. The profit and loss account and the balance sheet will be prepared as already explained.

Trading and Profit and Loss Account

It is the summary of such accounts which effect the profit or loss of the concern. These are prepared by transferring from the trial balance all nominal accounts and accounts relating to goods by means of journal entries called 'closing entries'. All remaining accounts i.e. real and personal, relating to properties, assets, debtors and creditors are shown in the balance sheet. In order to know the overall picture of the effect of these accounts they are grouped at one place. Items' increasing profit (revenue) are put on one side (credit) and those decreasing profits (losses and expenses) un the other side (debit). The balance is either net profit or net loss. This income statement is normally divided into two parts - first part is called trading account and second part is called profit and loss account. The procedure for preparing various accounts is discussed in details in the following pages:

Trading Account

As already discussed, first section of trading and profit and loss account is called trading account. The aim of preparing trading account is to find out gross profit or gross loss while that of second section is to find out net profit or net loss.

Preparation of Trading Account

Trading account is prepared mainly to know the profitability of the goods bought (or manufactured) sold by the businessman. The difference between selling price and cost of goods sold is the,5 earning of the businessman. Thus in order to calculate the gross earning, it is necessary to know:

(a) cost of goods sold.
(b) sales.

Total sales can be ascertained from the sales ledger. The cost of goods sold is, however, calculated. n order to calculate the cost of sales it is necessary to know its meaning. The 'cost of goods' includes the purchase price of the goods plus expenses relating to purchase of goods and brining the goods to the place of business. In order to calculate the cost of goods " we should deduct from the total cost of goods purchased the cost of goods in hand. We can study this phenomenon with the help of following formula:

Opening stock + cost of purchases - closing stock = cost of sales

As already discussed that the purpose of preparing trading account is to calculate the gross profit of the business. It can be described as excess of amount of 'Sales' over 'Cost of Sales'. This definition can be explained in terms of following equation:

Gross Profit = Sales-Cost of goods sold or (Sales + Closing Stock) -(Stock in the beginning + Purchases + Direct Expenses) 

The opening stock and purchases along with buying and bringing expenses (direct exp.) are recorded the debit side whereas sales and closing stock is recorded on the credit side. If credit side isJeater than the debit side the difference is written on the debit side as gross profit which is ultimately recorded on the credit side of profit and loss account. When the debit side exceeds the credit side, the difference is gross loss which is recorded at credit side and ultimately shown on the debit side of profit & loss account.

Usual Items in a Trading Account:

A) Debit Side

1. Opening Stock. It is the stock which remained unsold at the end of previous year. It must have been brought into books with the help of opening entry; so it always appears inside the trial balance. Generally, it is shown as first item at the debit side of trading account. Of course, in the first year of a business there will be no opening stock. 
2. Purchases. It is normally second item on the debit side of trading account. 'Purchases' mean total purchases i.e. cash plus credit purchases. Any return outwards (purchases return) should be deducted out of purchases to find out the net purchases. Sometimes goods are received before the relevant invoice from the supplier. In such a situation, on the date of preparing final accounts an entry should be passed to debit the purchases account and to credit the suppliers' account with the cost of goods.
3. Buying Expenses. All expenses relating to purchase of goods are also debited in the trading account. These include-wages, carriage inwards freight, duty, clearing charges, dock charges, excise duty, octroi and import duty etc.
4. Manufacturing Expenses. Such expenses are incurred by businessmen to manufacture or to render the goods in saleable condition viz., motive power, gas fuel, stores, royalties, factory expenses, foreman and supervisor's salary etc.
Though manufacturing expenses are strictly to be taken in the manufacturing account since we are preparing only trading account, expenses of this type may also be included in the trading account.

(B) Credit Side

1. Sales. Sales mean total sales i.e. cash plus credit sales. If there are any sales returns, these should be deducted from sales. So net sales are credited to trading account. If an asset of the firm has been sold, it should not be included in the sales.
2. Closing Stock. It is the value of stock lying unsold in the godown or shop on the last date of accounting period. Normally closing stock is given outside the trial balance in that case it is shown on the credit side of trading account. But if it is given inside the trial balance, it is not to be shown on the credit side of trading account but appears only in the balance sheet as asset. Closing stock should be valued at cost or market price whichever is less.

Valuation of Closing Stock

The ascertain the value of closing stock it is necessary to make a complete inventory or list of all the items in the god own together with quantities. On the basis of physical observation the stock lists are prepared and the value of total stock is calculated on the basis of unit value. Thus, it is clear that stock-taking entails (i) inventorying, (ii) pricing. Each item is priced at cost, unless the market price is lower. Pricing an inventory at cost is easy if cost remains fixed. But prices remain fluctuating; so the valuation of stock is done on the basis of one of many valuation methods.

The preparation of trading account helps the trade to know the relationship between the costs be incurred and the revenues earned and the level of efficiency with which operations have been conducted. The ratio of gross profit to sales is very significant: it is arrived at :

Gross Profit X 100 / Sales

With the help of G.P. ratio he can ascertain as to how efficiently he is running the business higher the ratio, better will be the efficiency.

Closing Entries pertaining to trading Account

For transferring various accounts relating to goods and buying expenses, following closing entries recorded:

(i) For opening Stock: Debit trading account and credit stock account
(ii) For purchases: Debit trading account and credit purchases account, the amount being the et amount after deducting purchases returns.
(iii) For purchases returns: Debit purchases return account and credit purchases account.
(iv) For returns inwards: Debit sales account and credit sales return account
(v) For direct expenses: Debit trading account and credit direct expenses accounts individually.
(vi) For sales: Debit sales account and credit trading account. We will find that all the accounts as mentioned above will be closed with the exception of trading account
(vii) For closing stock: Debit closing stock account and credit trading account After recording above entries the trading account will be balanced and difference of two sides ascertained. If credit side is more the result is gross profit for which following entry is recorded.
(viii) For gross profit: Debit trading account and credit profit and loss account If the result is gross loss the above entry is reversed.

Profit and Loss Account

The profit and loss account is opened by recording the gross profit (on credit side) or gross loss (debit side).

For earning net profit a businessman has to incur many more expenses in addition to the direct expenses. Those expenses are deducted from profit (or added to gross loss), the resultant figure will be net profit or net loss.

The expenses which are recorded in profit and loss account are ailed 'indirect expenses'. These be classified as follows:

Selling and distribution expenses. 

These comprise of following expenses: 

(a) Salesmen's salary and commission 
(b) Commission to agents 
(c) Freight & carriage on sales 
(d) Sales tax 
(e) Bad debts 
(j) Advertising 
(g) Packing expenses 
(h) Export duty

Administrative Expenses. 
These include: 

(a) Office salaries & wages 
(b) Insurance 
(c) Legal expenses 
(d) Trade expenses 
(e) Rates & taxes 
(f) Audit fees 
(g) Insurance 
(h) Rent 
(i) Printing and stationery 
(j) Postage and telegrams 
(k) Bank charges

Financial Expenses
These comprise: 

(a) Discount allowed
(b) Interest on Capital
(c) Interest on loan
(d) Discount Charges on bill discounted

Maintenance, depreciations and Provisions etc. 
These include following expenses

(a) Repairs 
(b) Depreciation on assets 
(c) Provision or reserve for doubtful debts 
(d) Reserve for discount on debtors.

Along with above indirect expenses the debit side of profit and loss account comprises of various business losses also.

On the credit side of profit and loss account the items recorded are: 

(a) Discount received 
(b) Commission received 
(c) Rent received 
(d) Interest received 
(e) Income from investments 
(j) Profit on sale of assets 
(g) Bad debts recovered 
(h) Dividend received 
(i) Apprenticeship premium etc.

Preparation of Profit and Loss Account

As already stated profit and loss account is commenced with gross profit or gross loss as ascertained by trading account. Then the profit and loss account is debited with all indirect expenses and losses. This results in closing of indirect expenses and losses account. The profit and loss account is then credited with various incomes and gains accounts by which all these accounts are closed.

Explanation of Certain items of Profit and Loss Account

1. Salaries

Salaries are paid for the services of employees and are debited to profit and loss ac- count being indirect expense. If any salary has been paid to proprietor or partners, it should be shown separately because it requires special treatment at the time of income tax assessment.

2. Salaries and Wages

When wages account is included with salaries it treated is as indirect expense and is taken into profit and loss account.

3. Rent

Rent of the office shop showroom or godown is an indirect expense and so is debited to profit & loss account. However, rent of factory is debited to trading account. When a part of the building has been sublet the rent received should be shown on the credit side of profit and loss account as a separate item.

4. Rates and Taxes

These are levied by the local authorities to meet public expenditure. It being an indirect expenditure is shown on the debit side of profit and loss account.

5. Interest

Interest on loan, overdraft or overdue debts is payable by the firm. It is an indirect expense; so debited to profit and loss account. Interest on loan advanced by the firm on depositor investments is an income of the firm and so is credited to the profit and loss account.

If business has paid any interest on capital to its proprietor or partners it should also be debited in the profit and loss account but separately because this item needs special treatment at the time of income-tax assessment.

6. Commission

In business sometimes agents are appointed to effect sales, who are paid commission as their remuneration. So this being a selling expenses is shown on the debit side of profit and loss account. Sometimes commission is also paid on purchases of goods, such 'as expense should be debited in the trading account. Sometimes the firm can also act as an agent to the other business houses and in such cases it receives commission from them. Commission so received is shown on the credit side of profit and loss account.

7. Trade Expenses

They are also termed as 'sundry expenses'. Trade expenses represent expenses of such a nature for which it is not worthwhile to open separate accounts. Trade expenses are not taken to trading account.

8. Repairs

Repairs to the plant, machinery, building are indirect expenses are treated expense and are debited to profit and loss account..

9. Traveling Expenses

Unless mentioned otherwise, traveling expenses are treated as indirect expenses and are debited to profit and loss account.

10. Horse & stable Expenses

Expenses incurred for the fodder of horses and wages paid for looking after stable are treated as indirect expenses and debited to profit and loss account.

11. Apprentice Premium

This is the amount charged from persons to whom training is imparted by the business. It is an income and is credited to profit and loss account. In case apprentice premium is charged in advance for two or three years, then the amount is distributed over number of years and each year's profit and loss account is credited with its share of income.

12. Bad debts

It is the amount which could not be recovered by the trader on account of credit sales. It is a business loss, so is debited in the profit and loss account.

13. Life Insurance Premium

If the premium is paid on the life policy of the proprietor of the business; it is treated as his drawings and is shown by way of deduction from the capital account. It should not be taken to profit and loss account.

14. Insurance Premium

If insurance premium account appears in the trial balance, it stands for the insurance of the business. This is taken to profit and loss account. Insurance premium on goods purchased, factory building, factory machines are treated as direct expense and are taken to trading account.

15. Income Tax

In the case of merchant income-tax paid is treated as a personal expense and is shown by way of deduction from capital account. Income-tax in case of companies is treated differently.

16. Discount allowed and Received

Discount is a reward for prompt payment. It is belief to show discount received and discount allowed separately on the credit and debit side of profit and loss account respectively instead of showing the net balance of this account.

17. Depreciation

Depreciation is a loss incurred on account of use of fixed assets in the business. Generally, it is charged from profit and loss account at a fixed percentage. The students should exercise great care as regards the rate of depreciation. If rate is without words 'per annum', then the rate will be taken irrespective of the period of accounts. This is very important when the period of accounts is less than one year. On the other hand, if the rate of depreciation is 'per annum' the depreciation should be calculated on the assets with due consideration to the period for which the asset has been used in business during the year. In case of additions to assets during the year, it is advisable to ignore depreciation on additions if the date of additions is not given. Same rule shall hold good for the sale of assets during the year.

18. Stock at the end appearing in the trial balance.

It is important to emphasize the rule that balance appearing in the trial balance is taken to one and only one place. It may either be trading account or profit and loss account or balance sheet. Since stock at the end is an asset, it will betaken to balance sheet. On the other hand, so long as there is stock in trade, account for that must be kept open and thus be taken to the assets side of balance sheet.

Adjustments of Final Accounts

The object of preparing final accounts is to find out the profits earned or losses suffered during a particular accounting period and to present the true picture of final position of the firm. While drawing the final accounts we have taken only those items of income and expenditure that are both earned and received and incurred and paid respectively.

To ensure that the final accounts disclose the true trading results, it is necessary to lake into account the whole of the expenses incurred, whether paid or not, and whole of the losses sustained. Likewise the incomes and gains earned, whether actually received or not, during the period covered by the trading and profit and loss account under consideration must also be recorded.

In mercantile system of accounting, it is essential to adjust different accounts before the preparation of final accounts. It is quite common to adjust expenses paid in advance, incomes received in advance, income accrued but not received, bad debts, provision for bad debts depreciation on assets and soon. Journal entries are passed to effect the required adjustments; these entries are known as adjusting entries.

Usual Adjustments
Outstanding Expenses:

Certain expenses relating to a particular period may not have been paid in that accounting period. All such expenses which are due for payment in one accounting year but actually paid in future accounting years or payment of which is postponed are all outstanding or unpaid expenses. All such expenses must be accounted for in that accounting year in which they are incurred, irrespective of the fact whether they are paid or not. In other words, all paid and also unpaid expenses must be recorded in an accounting year if they relate to that accounting year only with a view to ascertain true trading results e.g. if salaries for the last month are not paid, no entry will appear in books of accounts unless these are paid. So profit and loss account in respect of salaries will thus be under charged than the actual expenditure, therefore the profit will be more. 

Prepaid Expenses

The, benefit of some of the expenses already spent will be available in the next accounting year also, Such a portion of the expense is called pre-paid expense; since such expenses are already paid, they are also recorded in the books of accounts of that period to which they do not relate. The result shown by the final accounts of a particular period will not be correct because such expenses relate to future periods. Therefore, such prepaid expenses must be adjusted in the books of accounts to arrive at true profit. Generally insurance, taxes, telephone subscriptions, rent etc. are paid in advance, thus requiring adjustment e.g. Rent paid by x for one year on 1.7.79 when his accounting year is calendar year; thus rent for 6 months will remain UN-exhausted and will be c/f to the next year.

Accrued Income

There may be certain incomes which have been earned during the year but not yet received till the end of the year. Income like interest on investments, rent and commission etc. are normally earned by merchant during a particular accounting period but actually not received during that period. Such income items need adjustments before the preparation of final accounts. Such incomes should be credited to that particular income account. At the same time the income so -earned but not received is an asset because the amount is still to be received.

Income Received in Advance

Sometimes, traders receive certain amounts during a particular trading period which are to be earned by them in future periods. Such incomes though actually received and therefore, recorded i.e. not yet earned. Such incomes should be credited to the profit and loss account of the year in which these are earned. Therefore, such income though received is not the income but a liability of that period

Closing Stock

It represents the unsold stock at the end of the year. Closing stock is valued and following entry is passed at the end of the year: Closing Stock account To Trading Account Closing stock at the end appears in the balance sheet and is carried forward to the next year. At the end of the next year it appears in the trial balance as opening stock and from there it is taken to debit side of trading account and thus closed.

Depreciation

The value of fixed assets diminishes gradually with their use for business purposes. Although this decrease in the value happens every day but its accounting is done only at the end of accounting period with the help of following entry :Depreciation account To Particulars asset

Interest on Capital

The proprietor may wish to ascertain his profit after considering the interest which he losses by investing his money in the firm. Interest to be charged is an expense for the business on one hand and income to the proprietor on the other hand. Following adjusting entry is recorded at the end of accounting period: Interest on capital a/e To Capital a/c Interest on capital being an expense is debited to profit and loss account and same amount of interest on capital is added to capital.

Interest on Drawings

As business allows interest on capital it also charges interest on drawings made by the proprietor. Interest so charged is an income for the business on one hand and expense for the proprietor on the other hand. Following adjusting entry is passed at the end. of accounting period: Capital ale Dr. To Interest on drawings a/e The interest on drawings being an income is credited to profit and loss account is shown as a deduction from the capital.

Bad Debt to be written off

Bad debts are irrecoverable debts from customers, during the course of the financial year. These are recorded as follows: Bad debts a/c To Sundry Debtors a/c It results in the reduction of customers debit balance and addition to the loss i.e. Bad Debts. At the end of the year when the trial balance is drawn, these two accounts show debit balances. The balance on sundry debtors account, thus arrived, is the net balance, after deduction of any bad debts recorded during the year. But after the trial balance is prepared and before the final accounts are drawn trader may find that there are additional bad debts. Such bad debts must be recorded with the same adjusting entry and giving it following effect in ledger and final accounts.

Provision for Bad Debts

At the end of the year, after writing off the bad debts about whom we were sure of becoming irrecoverable, there may still be some customer balances from whom it is doubtful to collect the entire amount. However, it cant be written off as bad because non-recovery of such amount is not certain. But at the same time the balance in sundry debtors account should be brought down to its net realizable figure so that balance sheet may not exhibit the debtors at more than their actual realizable value. Therefore, to show the approximately correct value of the sundry debtors in the balance sheet a provision or reserve is created for possible bad debts. Such an adjustment entry is recorded at the end of accounting year.

Provision for bad debts is an attempt to anticipate possible losses due to bad debts and to keep aside an amount out of profit to meet the loss estimated in the following years. When the provision for bad debts is created, following entry is recorded:
Profit and Loss A/c Dr. To Provision for bad debts A/c

Some important considerations while creating provision for bad debts

(i) Sundry debtors account should not be credited with the amount of provision for doubtful debts because the loss has not actually been incurred.
(ii) Treatment of bad debts or provision for bad debts appearing inside the trial balance. If some balance (credit) is already appearing in provision for doubtful debts account inside the trial balance, it is the previous years UN-utilized balance of this account. If some bad debts are also appearing on the debit side of the trial balance, these should be transferred to provision for bad debts account, with the help of following entry: Provision for bad debts a/e To Bad debts a/e. It is important to note that, as these items appear inside the trial balance, so these are to appear only in profit and loss account as debtors have already been reduced during the year.
(iii) When bad debts and provision for bad debts appear in trial balance, new provision is to be created and further bad debts are to be written off. If already bad debts and provision for bad debts are appearing in trial balance, these should be adjusted and only difference should be taken to profit and loss account.

If bad debts written off plus bad debts to be written off plus new provision for bad debts is more than the credit balance of old provision appearing in the trial balance, the difference should be debited to profit and loss account.

Provision for discount on Debtors

It is normal practice in trade to allow discount to customers for prompt payment and it constitutes a substantial sum. Sometimes the goods are sold on credit to customers in one accounting period where as the payment of the same is made by them in the next accounting period and so discount is to be allowed. It is a prudent policy to charge this expenditure to the period in which sales have been made, so a provision is created in the same manner, as in case of provision for doubtful debts

An important point to note is that no discount win be allowed on debts that become bad. Therefore, the provision required for discount will be in respect of the other debts only. So the amount of provision for discount be calculated after deducting the provision for bad debts from sundry debtors.

Provision for discount on creditors

Prompt payment, if made, enables a businessman to receive discount. The question arises whether this discount should be treated as income of the period in which purchases were made or of the period when the payment is made, if both events are in different accounting years, it has been well decided by accountants that it should be treated as income of the period in which purchases are made. So on last date of accounting period if some amount is still payable to creditors, a provision should be created for such probable income and amount should be credited to the profit and loss account of that year in which purchases are made. Following adjusting entry is passed for it :Provision for discount on creditors a/c Dr. To Profit and loss account

Losses by Accidents

Sometimes a business suffers certain losses not because of trading but because of certain accidents. These may destroy some fixed assets of the merchant. In such a case the asset account is credited and the profit and loss account is debited.

If goods (stock-in-trade) are lost by accident the value of closing stock win be lower than otherwise. This will reduce the amount of gross profit. So the cost of goods lost by accident is credited to the trading account and debited to the profit and loss account. The increase -in gross profit will be neutralized by the debit to the profit and loss account and thus the net profit will not be effected. The entries to the passed are as follows: Loss by accident a/c To Goods lost by accident a/c

Commission to manager payable on profits

Sometimes the manager is entitled to a commission on profits.. Such commission may be :

(a) Fixed percentage on net profits before charging such commission.
(b) Fixed percentage on net profits- after charging such commission.

Such commission being an expense is debited to commission account. However, as it has not yet been paid, so commission payable account is given the credit and finally it is shown in the balance sheet as a liability. Calculation of Commission First of all trading account should be prepared in usual manner and after transferring the gross profit or loss all expenses and incomes should be debited or credited except the commission which is still to be calculated.

Goods used in business

Sometimes goods purchased for the purpose of resale are used in business as giving them away for charitable purpose or distributing them as free samples. In these conditions purchases account should be credited with an amount equal to the cost of goods used in business and same amount is debited to charity or advertisement expenses account, as the case may be.

**Comment by lcabrini** [comment]
Wow, this was a lot of text. If this is background to accounting, couldn't we just provide a link or maybe put the text in the wiki. The issue tracker should be used to track issues specifically.

**Comment by BorderITS** [comment]
it wasn't from one page i did some digging to combine all this

**Comment by lcabrini** [comment]
OK, that's fine. But what specific issue is this? An issue should be about one specific functionaliy, data model or UI. This is a lot of text and I cannot make out exactly what this is about.

**Comment by BorderITS** [comment]
but we should still mange to read to understand because reasons why i posted this is , it will help us to understand the structure of the accounting

**Comment by lcabrini** [comment]
that is fine, just that maybe this would be better in the wiki. In the issue tracker, it indicates something that needs to be done.

**Comment by BorderITS** [comment]
so do you want me to removed it ? 
And  put it into wiki
